plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/AbstractTextPanelEditor.java

```diff
@@ -269,44 +287,71 @@ public <T> T getAdapter(Class<T> adapter) {
     }
 
     @Override
-    public void primeEditorValue(@NotNull DBRProgressMonitor monitor, @NotNull StyledText control, @Nullable DBDContent value) throws DBException
-    {
+    public void primeEditorValue(@NotNull DBRProgressMonitor monitor, @NotNull StyledText control,
+            @Nullable DBDContent value) throws DBException {
         try {
-            // Load contents in two steps (empty + real in async mode). Workaround for some strange bug in StyledText in E4.13 (#6701)
-            final TextViewer textViewer = editor.getTextViewer();
-            final String encoding = getPanelSettings().get(PREF_TEXT_EDITOR_ENCODING);
-            final ContentEditorInput textInput = new ContentEditorInput(valueController, null, null, encoding, monitor);
-            boolean longContent = textInput.getContentLength() > LONG_CONTENT_LENGTH;
-            if (longContent) {
-                UIUtils.asyncExec(() -> {
-                    editor.setInput(new StringEditorInput("Empty", "", true, StandardCharsets.UTF_8.name()));
-                });
+            int maxContentSize = DBWorkbench.getPlatform().getPreferenceStore()
+                    .getInt(ModelPreferences.EDITING_CONTENT_MAX_SIZE_KBYTES) * 1000;
```

Comment:
Also we don't use 8 spaces for indentation in such places. Always 4 spaces :)
```suggestion
            int maxContentSize = DBWorkbench.getPlatform().getPreferenceStore()
                .getInt(ModelPreferences.EDITING_CONTENT_MAX_SIZE_KBYTES) * 1000;
```

Reply:
Already reformatted